### PR TITLE
Add Open RSC

### PR DESCRIPTION
**Pull request by dginovker**
Open RSC is a remake of the original version of Runescape, circa ~2001-2003.

### CODE REVIEW

File: originals/r.yaml [hunk 1]
```diff
@@ -142,6 +142,15 @@
...
     - RPG
 
+- name: [Runescape Classic, 'https://classic.runescape.wiki/']
+  names:
+  - Runescape Classic
+  - RSC
+  meta:
+    genre: 
+    - MMORPG
+    - Aventure
```
Comment (cxong) [inline originals/r.yaml]:
```suggestion
    - Adventure
```

File: originals/r.yaml [hunk 2]
```diff
@@ -142,6 +142,15 @@
     genre:
     - RPG
 
+- name: [Runescape Classic, 'https://classic.runescape.wiki/']
+  names:
+  - Runescape Classic
+  - RSC
```
Comment (cxong) [inline originals/r.yaml]:
```suggestion
  - RSC
  external:
    website: 'https://classic.runescape.wiki/'
```

File: originals/r.yaml [hunk 3]
```diff
@@ -142,6 +142,15 @@
     genre:
     - RPG
 
+- name: [Runescape Classic, 'https://classic.runescape.wiki/']
```
Comment (cxong) [inline originals/r.yaml]:
```suggestion
```

File: games/r.yaml [hunk 4]
```diff
@@ -438,6 +438,28 @@
   type: remake
   updated: 2015-05-15
 
+- name: Open RSC
+  type: Remake
```
Comment (cxong) [inline games/r.yaml]:
```suggestion
  type: remake
```

**Comment by dginovker** [comment]
Ready to merge

**Pull request merged**